README.md

```diff
@@ -6,6 +6,24 @@ exports mumble / murmur server stats by using the mumble ping protocol
 
 The exporter expects a single HTTP parameter named `host` that must include a port, i.e. `mumble.example.com:1337`.
 
+You can test if your the exporter works correctly with
+```
+curl localhost:8778?host=mumble.example.com:63748
+```
+
+Prometheus job example
+```
+scrape_config
+ - job_name: 'mumble'
+    scrape_interval: 120s
+    scheme: http
+    params:
+      host: ['mumble.example.com:64738']
+    static_configs:
+      - targets:
+        - 'localhost:8778'
```

Comment:
For these kinds of exporters, a blackbox-like scrape_config with relabelings IMHO makes more sense:

```
scrape_configs:
  - job_name: 'mumble'
    static_configs:
      - targets:
        - mumble.example.com:64738
        - mumble.acme.com:64738
    relabel_configs:
      - source_labels: [__address__]
        target_label: __param_target
      - source_labels: [__param_target]
        target_label: instance
      - target_label: __address__
        replacement: localhost:8778
```

Reply:
used improvement in [bbc92e2](https://github.com/mguentner/mumble_exporter/pull/2/commits/bbc92e2d8c168013eab84aa5b79d8cc5d857ffe5)